SocialFilter/sections/specific.txt

```diff
@@ -6328,6 +6328,7 @@ animal.mx###ax_single_bar_sharers
 allcalidad.la###barraPlus1
 tecmundo.com.br##.tec--article__share-grid
 tecmundo.com.br##.tec--share
+artstation.com##.share-grid > .s-grid
```

Comment:
```suggestion
artstation.com##.share-grid
```
is more optimal

Reply:
It doesn't work in apps - https://github.com/AdguardTeam/AdguardFilters/pull/177830#discussion_r1578145084